dipy/viz/ui.py

```diff
@@ -60,10 +54,15 @@ class UI(object):
     """
 
     def __init__(self):
```

Comment:
I agree that it would be nice to have these in `__init__`, but if it's a problem then the other way is fine. I'm ready to approve when these are removed.

Reply:
I did the refactor and created a new PR #1492